### PR TITLE
Update Rocket pings to 2020 Jan 21st

### PR DESCRIPTION
**Pull request by Conjuror**

### CODE REVIEW

File: templates/telemetry/rocket_android_events_v1/view.sql [hunk 1]
```diff
@@ -32,14 +32,12 @@ SELECT
         WHEN (event_category IN ('action') ) AND (event_method IN ('launch') ) AND (event_object IN ('app') ) AND (event_value IN ('shortcut') ) THEN 'Rocket -  App is launched by Shortcut' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('launch') ) AND (event_object IN ('app') ) AND (event_value IN ('private_mode') ) THEN 'Rocket -  App is launched from Private Shortcut' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('show') ) AND (event_object IN ('private_shortcut') ) THEN 'Rocket -  Show private shortcut prompt' 
-        WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('private_shortcut') ) AND (event_value IN ('[positive|negative|dismiss]') ) THEN 'Rocket -  Click private shortcut prompt' 
+        WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('private_shortcut') ) AND (event_value IN ('positive', 'negative', 'dismiss') ) THEN 'Rocket -  Click private shortcut prompt'
```
Comment (akkomar) [inline templates/telemetry/rocket_android_events_v1/view.sql]: Good catch!

**Comment by Conjuror** [comment]
Commit log is revised accordingly. Thank you for kindly help, @akkomar

**Comment by yshlin** [comment]
Sorry I'm having problem login to my github,
so I'm posting w/t another ID for backfill events

- Backfill from 2019/6/11
   - Rocket - Click private shortcut prompt
- Backfill from 2019/11/4
   - Rocket - Click Toolbar - Share
   - Rocket - Click Toolbar - Open in browser
   - Rocket - Click Toolbar - Back
   - Rocket - Show Game Shortcut Contextual Hint
- Backfill from 2019/12/25
   - Rocket - Select Query Content Home
   - Rocket - Show Content Home Search Bar
   - Rocket - Open Detail Page More
   - Rocket - Change Travel Settings
   - Rocket - Click Content Home Item
   - Rocket - Show Travel Search Result Message
   - Rocket - Click Travel Search Result Message
   - Rocket - Show Set-default Travel Search Message
   - Rocket - Click Set-default Travel Search Message
- Backfill from 2020/1/15
   - Rocket - Dismiss D1 Notification
   - Rocket - Open D1 Notification

**Comment by yshlin** [comment]
Found one problem in the mapping script that may cause one missing event.
May need to update before merge

**Pull request merged**